Client/src/Components/Inputs/TextInput/Adornments/index.jsx

```diff
@@ -23,14 +25,15 @@ export const HttpAdornment = ({ https }) => {
 				color={theme.palette.text.secondary}
 				sx={{ lineHeight: 1, opacity: 0.8 }}
 			>
-				{https ? "https" : "http"}://
+				{prefix !== undefined ? prefix : https ? "https://" : "http://"}
```

Comment:
What's the purpose of adding the `prefix` prop here?  I'm not sure I understand the use case, could you elaborate?